extensions/wikia/AdEngine/js/video/googleIma.js

```diff
@@ -0,0 +1,74 @@
+/*global define, google*/
+define('ext.wikia.adEngine.video.googleIma', [
+	'ext.wikia.adEngine.video.vastBuilder',
+	'wikia.lazyqueue',
+	'ext.wikia.adEngine.utils.scriptLoader'
+], function (vastBuilder, lazyQueue, scriptLoader) {
+	'use strict';
+	var adDisplayContainer,
+		adsLoader,
+		adsManager,
+		imaLibraryUrl = '//imasdk.googleapis.com/js/sdkloader/ima3.js',
+		isAdsManagerLoaded = false,
+		videoMock = document.createElement('video');
+
+	function initialize() {
```

Comment:
Is there a reason we use initialize() instead of init()? In most cases in AdEngine we use 'init', it would be nice to have it consistent.